### ISSUE TITLE
[QnA] Naming Conventions

### ISSUE THREAD

**Issue by justjjasper** [Nugget-Reactor/FEC#65]
https://github.com/Nugget-Reactor/FEC/blob/0e5934a3bb49fa218f840e9b56deafb12b011081/client/src/Components/QnA/QuestionsList.jsx#L22-L42

Is the naming Convention “More Answers Questions” practical? Some questions that are being loaded do not have answered questions.

**Comment by cwang1014** [comment]
I agree that was a bad name that I pulled directly from brd, renamed to MoreQuestions.
